### PR TITLE
api changes around registartion and open order

### PR DESCRIPTION
**Pull request by doodyparizada**

### CODE REVIEW

File: scripts/src/mock_client.ts [hunk 1]
```diff
@@ -18,10 +18,12 @@ import {
 	CollectionPage
 } from "stellar-sdk";
 import { CompletedPayment } from "./internal/services";
+import { SpendPayloadOffer } from "./public/services/applications";
 
-// const BASE = "http://localhost:3000";
-const BASE = "https://api.kinmarketplace.com"; // production - XXX get this from env var?
-const APP_ID = "smpl";
+const BASE = "http://localhost:3000";
+const SAMPLE_BASE = "http://localhost:3002";
```
Comment (nitzantomer) [inline scripts/src/mock_client.ts]: maybe `JWT_SERVICE_BASE`

File: scripts/src/models/orders.ts [hunk 2]
```diff
@@ -101,7 +101,7 @@ export class Order extends CreationDateModel {
 	public type!: OfferType;
 
 	@Column("simple-json", { name: "blockchain_data", nullable: true })
-	public blockchainData?: BlockchainData;
+	public blockchainData!: BlockchainData;
```
Comment (nitzantomer) [inline scripts/src/models/orders.ts]: why?
it can be nullable based on the column meta

File: scripts/src/public/jwt.ts [hunk 3]
```diff
@@ -3,30 +3,36 @@ import * as jsonwebtoken from "jsonwebtoken";
 import { Application } from "../models/applications";
 
 export type JWTClaims = {
-	iss: string; // issuer
+	iss: string; // issuer - the app_id
 	exp: number; // expiration
 	iat: number; // issued at
 	sub: string; // subject
-};
+	};
```
Comment (nitzantomer) [inline scripts/src/public/jwt.ts]: what's this indentation all about?

File: scripts/src/models/orders.ts (hunk 2 above)
Comment (doodyparizada) [reply scripts/src/models/orders.ts]: what? now we always set the blockchain data on both marketplace and external orders

Comment (doodyparizada) [reply scripts/src/models/orders.ts]: same def as Offer table

**Comment by doodyparizada** [comment]
@nitzantomer approved

**Pull request merged**